test/index.js

```diff
@@ -76,7 +77,7 @@ lab.experiment('Connections', () => {
         lab.expect(db).to.be.an.instanceof(Mongodb.Db);
         lab.expect(db.serverConfig.isConnected()).to.equal(true);
 
-        class DummyModel extends MongoModels {};
+        class DummyModel extends MongoModels {};    // eslint-disable-line
```

Comment:
What's the lint error? Can we fix without using the comment?

Reply:
"padding-line-between-statements - Expected blank line before this statement."

Turns out if I drop the semicolon it doesn't complain.